src/org/etools/j1939_84/bus/j1939/J1939TP.java

```diff
@@ -241,50 +225,59 @@ private void receive(Packet packet) {
             }
             // everything else, pass through
             inbound.send(packet);
+            // } else {
+            // System.err.println("ignoring:" + packet);
```

Comment:
Commented out code should be removed

Reply:
I need to know how to add debug logging.

Reply:
I will work out add a CustomLogger that will write to the existing log and also to StdOut/Err